mlflow/tracking/fluent.py

```diff
@@ -830,6 +830,29 @@ def set_experiment_tag(key: str, value: Any) -> None:
     MlflowClient().set_experiment_tag(experiment_id, key, value)
 
 
+def delete_experiment_tag(key: str) -> None:
+    """
+    Delete a tag from the current experiment.
+
+    Args:
+        key: Tag name. This string may only contain alphanumerics, underscores (_), dashes (-),
+            periods (.), spaces ( ), and slashes (/). All backend stores will support keys up to
+            length 250, but some may support larger keys.
+
+    .. code-block:: python
+        :test:
+        :caption: Example
+
+        import mlflow
+
+        mlflow.set_experiment("test-delete-tag")
+        mlflow.set_experiment_tag("release.version", "1.0")
+        mlflow.delete_experiment_tag("release.version")
```

Comment:
```suggestion
        mlflow.delete_experiment_tag("release.version")
        asssert ...
```

can we add `assert` to ensure it's really deleted?